src/pretix/plugins/sendmail/models.py

```diff
@@ -278,16 +279,26 @@ def save(self, **kwargs):
 
         create_sms = []
         if self.event.has_subevents:
-            for se in self.event.subevents.annotate(has_sm=Exists(ScheduledMail.objects.filter(
-                    subevent=OuterRef('pk'), rule=self))).filter(has_sm=False):
-                sm = ScheduledMail(rule=self, subevent=se, event=self.event)
-                sm.recompute()
-                create_sms.append(sm)
+            if self.subevent:
+                ScheduledMail.objects.get_or_create(rule=self, subevent=self.subevent, event=self.event)
+            else:
+                for se in self.event.subevents.annotate(has_sm=Exists(ScheduledMail.objects.filter(
+                        subevent=OuterRef('pk'), rule=self))).filter(has_sm=False):
+                    sm = ScheduledMail(rule=self, subevent=se, event=self.event)
+                    sm.recompute()
+                    create_sms.append(sm)
             ScheduledMail.objects.bulk_create(create_sms)
         else:
             ScheduledMail.objects.get_or_create(rule=self, event=self.event)
 
         if not is_creation:
+            if self.subevent:
+                keep_states = [ScheduledMail.STATE_COMPLETED,
+                               ScheduledMail.STATE_FAILED]  # we keep rules that have already been sent at least partially
```

Comment:
I just read through ``recompute()`` and I believe COMPLETED is the only state where we have no risk of the rule running ever again.

```suggestion
                keep_states = [ScheduledMail.STATE_COMPLETED]  # we keep rules that have already been sent
```